### PR TITLE
Avtar

### PR DESCRIPTION
**Pull request by SumitSaundadkar**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -144,11 +146,35 @@
...
+      height: 1rem;
+      position: absolute;
+      background-color: red;
+      border-radius: 50%;
+      margin-top: -1.4em;
+      margin-left: 3.5rem;
+  }
+   .status-online{
+      border: 2px solid green;
+      background-color: green;
```
Comment (horsemaker) [inline style.css]: You could use color variables here, instead of specifying green.

File: style.css [hunk 2]
```diff
@@ -144,11 +146,35 @@
...
+
+  }
+  .status-online-offline{
+      border: 2px solid rgb(15, 17, 15);
+      background-color: rgb(200, 214, 200);
```
Comment (horsemaker) [inline style.css]: Same here, the use of color variables would be beneficial.

File: style.css [hunk 3]
```diff
@@ -144,11 +146,35 @@
...
     margin-top: 1rem;
     display: flex;
     justify-content: center;
   }
+
+  .status{
+      width: 1rem;
+      height: 1rem;
+      position: absolute;
+      background-color: red;
```
Comment (horsemaker) [inline style.css]: use color variables

**Comment by rahulyadav139** [comment]
Everything is great but badge placement on the avatar is a bit weird side especially for the large one. You should fix that.

File: style.css (hunk 2 above)
Comment (SumitSaundadkar) [reply style.css]: thanks for the review.

File: style.css (hunk 3 above)
Comment (SumitSaundadkar) [reply style.css]: updated

File: index.html [hunk 4]
```diff
@@ -79,23 +79,77 @@ <h1>Avatar</h1>
...
-                style="
-                  width: 555px;
-                  height: 409px;
-                  border: 0;
-                  transform: scale(1);
-                  overflow: hidden;
-                "
+                style="width: 766px; height: 502px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (aiman-ansari) [inline index.html]: Instead of using inline css make class and give styling to that class
And use it inside of iframe.

File: index.html [hunk 5]
```diff
@@ -79,23 +79,77 @@ <h1>Avatar</h1>
...
+            style="width: 766px; height: 502px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (aiman-ansari) [inline index.html]: Same here

**Review by aiman-ansari — CHANGES_REQUESTED** [review]
Your avatar will look good if you give either border or shadow to it

File: components/avtar.css [hunk 6]
```diff
@@ -1,17 +1,27 @@
...
   }
   .avatar-md img {
-    width: 5rem;
+    width: 7rem;
     border-radius: 50%;
   }
   .avatar-l img {
     width: 7rem;
     border-radius: 50%;
   }
```
Comment (vinaymahamuni) [inline components/avtar.css]: Medium and large image have same width. It should be different.

File: components/avtar.css [hunk 7]
```diff
@@ -1,17 +1,27 @@
 .avatar-sm img {
```
Comment (vinaymahamuni) [inline components/avtar.css:1]: Suggestion: You can use prettier plugin to format the html and css document. It helps in reading code easily.

File: components/avtar.css [hunk 8]
```diff
@@ -1,17 +1,27 @@
...
+  /* .status-online{
+    border: 2px solid green;
+    background-color: green;
+
+}
+.status-online-offline{
+    border: 2px solid rgb(15, 17, 15);
+    background-color: rgb(200, 214, 200);
+
+} */
```
Comment (vinaymahamuni) [inline components/avtar.css]: You can remove commented code if its not going to be used.

File: index.html (hunk 4 above)
Comment (vinaymahamuni) [inline index.html]: This style is duplicated in next iframe as well. What can be the solution to avoid this duplication?

File: index.html [hunk 9]
```diff
@@ -79,23 +79,77 @@ <h1>Avatar</h1>
...
+                    <img src="./img/hacker.png" alt="avatar-medium" />
+                    <div class="status status-online"></div>
+                  </div>
+                  <div class="avatar-l">
+                    <img src="./img/hacker.png" alt="avatar-large" />
+                    <div class="status status-online-offline"></div>
+                  </div>
+                  <!-- <div class="avatar-xl">
+                    <img src="./img/hacker.png"alt="avatar-extraLarge" />
+                  </div> -->
```
Comment (vinaymahamuni) [inline index.html:120]: This code can be deleted if not getting used.

File: index.html [hunk 10]
```diff
@@ -79,23 +79,77 @@ <h1>Avatar</h1>
...
+        
+
+
+        
+
+    
+        
+
+
+
```
Comment (vinaymahamuni) [inline index.html:144]: unnecessary extra lines

File: index.html [hunk 11]
```diff
@@ -79,23 +79,77 @@ <h1>Avatar</h1>
...
+        
       </main>
+
+
+      
+
+
+
+
```
Comment (vinaymahamuni) [inline index.html:152]: unnecessary extra lines

File: style.css [hunk 12]
```diff
@@ -2,14 +2,15 @@
 @import url(/components/avtar.css);
 
 :root {
-    --cta-color: #0e0e0f;
+    --cta-color: #05051a;
     --hover-bg: #8484c7;
     --primary-color: #161617;
-    --secondary-color: #5d585d;
+    --secondary-color: #441744;
```
Comment (vinaymahamuni) [inline style.css:8]: nice use of variables in css  👍

File: style.css [hunk 13]
```diff
@@ -2,14 +2,15 @@
...
     --success-color: #3c8855;
     --warning-color: #f5c118;
     --error-color: #dc3545;
     --info-color: #5dcaf0;
+    --offline-color:#9dbac5;
   }
   
   * {
```
Comment (vinaymahamuni) [inline style.css:16]: Food for thought: When to use * and when to use 'body' or 'html'?

File: style.css [hunk 14]
```diff
@@ -144,11 +147,35 @@
...
     justify-content: center;
   }
+
+  .status{
+      width: 1rem;
+      height: 1rem;
+      position: absolute;
+      background-color:  var(--error-color);
+      border-radius: 50%;
+      margin-top: -1.4em;
```
Comment (vinaymahamuni) [inline style.css:179]: I think, by mistake, you have used em instead of rem

**Review by vinaymahamuni — COMMENTED** [review]
Please check the comments

File: style.css [hunk 15]
```diff
@@ -2,14 +2,15 @@
 @import url(/components/avtar.css);
 
 :root {
-    --cta-color: #0e0e0f;
+    --cta-color: #05051a;
```
Comment (tarunsankhla) [inline style.css:5]: You can create variable name for color hex

File: style.css [hunk 16]
```diff
@@ -144,11 +147,35 @@
...
+      margin-top: -1.4em;
+      margin-left: 4.5rem;
+  }
+   .status-online{
+      border: 2px  ;
+      background-color: var(--success-color);
+
+  }
+  .status-online-offline{
+      border: 2px solid var(--primary-color) ;
```
Comment (tarunsankhla) [inline style.css:188]: Look great 👍

File: index.html (hunk 5 above)
Comment (SumitSaundadkar) [reply index.html]: updated

File: style.css (hunk 16 above)
Comment (SumitSaundadkar) [reply style.css:188]: thank you  tarun for the pr

**Comment by SumitSaundadkar** [comment]
merging avtar after pr

**Pull request merged**